### PR TITLE
Update OpenShift template versions

### LINKED ISSUE #1601: OpenShift Guide - Document RH-SSO 7.4.X stream of container images is already deprecated
### Describe the bug

The OpenShift guide of RH-SSO 7.5.2 version mentions RH-SSO 7.0 up to 7.3 as deprecated. And recommends the users to use either RH-SSO 7.4.X, or latest RH-SSO 7.5.2:
* https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.5/html-single/red_hat_single_sign-on_for_openshift/index#version_compatibility_and_support

But RH-SSO 7.4.X stream of container images is already deprecated in Red Hat Container Catalog:
* https://catalog.redhat.com/software/containers/rh-sso-7/sso74-openshift-rhel8/5e7e033d5a1346687697bbd3

thus this needs to be fixed.

### Version

main

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Close #1601

### CODE REVIEW

**Review by iankko — DISMISSED** [review]
LGTM Thanks, Andy!

File: topics/templates/document-attributes-product.adoc [hunk 1]
```diff
@@ -1,19 +1,20 @@
 :project_community: false
 :project_product: true
 :project_name: Red Hat Single Sign-On
-:project_versionMvn: 15.0.2.redhat-00001
-:project_versionNpm: 15.0.2.redhat-00001
+:project_versionMvn: 18.0.2.redhat-00001
```
Comment (miquelsi) [inline topics/templates/document-attributes-product.adoc]: 18.0.0.redhat-00001

File: topics/templates/document-attributes-product.adoc [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+:project_versionNpm: 18.0.2.redhat-00001
```
Comment (miquelsi) [inline topics/templates/document-attributes-product.adoc]: 18.0.0

**Comment by andymunro** [comment]
Thanks, @miquelsi Please review again and approve if okay.

**Review by miquelsi — APPROVED** [review]

**Pull request merged**